### PR TITLE
 Refactor code to get different question forms

### PR DESCRIPTION
**Pull request by mehtadarshan1**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -335,13 +335,22 @@ app.get('/questionform', function(req, res) {
...
+            break;
 
-app.get('/mc', function(req, res){
-    var html = multipleChoiceForm();
+        case 'mc':
+            html = multipleChoiceForm();
+            break;
+
+        default:
+            //callback('type is not defined', null);
```
Comment (esteeroramy) [inline server.js:350]: let's not keep coding comments in out codebase

File: server.js [hunk 2]
```diff
@@ -335,13 +335,22 @@ app.get('/questionform', function(req, res) {
...
+            res.status(400).send("No question Type defined");
```
Comment (esteeroramy) [inline server.js:351]: double quotes to single quotes please

**Review by esteeroramy — CHANGES_REQUESTED** [review]
Couple of minor fixes here. Also will this PR include reformatting the question structure as discussed in the last meeting?

File: server.js [hunk 3]
```diff
@@ -335,13 +335,22 @@ app.get('/questionform', function(req, res) {
     return res.status(200).send(html);
 });
 
-app.get('/re', function(req, res){
-    var html = regexForm();
-    res.status(200).send(html);
-});
+app.post('/questionFormAnswer', function(req, res){
```
Comment (ahmedqarmout2) [inline server.js:338]: hmmm i thought this is included in the other PR?!

File: server.js [hunk 4]
```diff
@@ -335,13 +335,22 @@ app.get('/questionform', function(req, res) {
...
+    var html;
+    switch (req.body.qType) {
+        case 're':
```
Comment (esteeroramy) [inline server.js:341]: changes here overlap with https://github.com/quizzard-edu/quizzard/pull/23/files

**Review by esteeroramy — CHANGES_REQUESTED** [review]

**Review by ahmedqarmout2 — CHANGES_REQUESTED** [review]

**Comment by esteeroramy** [comment]
If this PR is not valid anymore let's close it and just focus on the other one. Let us know why there is a duplication here please. Thanks :)

**Review by Mohamed-A-Mohamed — CHANGES_REQUESTED** [review]
Rejecting for now as this PR and PR #23 are working on similar things and I'm not sure which needs to be merged and which needs to be deleted.

**Review by Mohammed-JS-Khan — CHANGES_REQUESTED** [review]
As per the other comments, rejected due to conflicting changes with PR #23.

**Pull request closed without merging**